### ISSUE TITLE
error when you try to start trakx

### ISSUE THREAD

**Issue by a8underscore** [crimist/trakx#36]
```
nikita@localhost:~/Downloads/trakx> ./trakx start
starting...
failed to open log file: open /home/nikita/.cache/trakx/trakx.log: no such file or directory
```
go version go1.14.8 linux/amd64

**Comment by a8underscore** [comment]
when i create a /trakx/ folder in .cache it works fine

**Comment by a8underscore** [comment]
i cant reproduce this now

**Comment by crimist** [comment]
I've been having this issue as well. I believe it's a permissions issue and I'm looking into it.

**Comment by crimist** [comment]
Fixed: 1b7fd61db7a273c9c93b02839320921eb46593d2
